lms/djangoapps/courseware/module_render.py

```diff
@@ -754,10 +778,10 @@ def rebind_noauth_module_to_user(module, real_user):
             'cache': CacheService(cache),
             'sandbox': SandboxService(contentstore=contentstore, course_id=course_id),
             'xqueue': xqueue_service,
-            'replace_urls': replace_url_service
+            'replace_urls': replace_url_service,
+            'rebinder': rebinder_service,
```

Comment:
We should add it to `services` of the [XBlockRuntime](https://github.com/openedx/edx-platform/blob/732d8cb33771047a1c5f396f04c35f3279aafcd4/openedx/core/djangoapps/xblock/runtime/runtime.py#L254-L255).